cli/azd/docs/environment-variables.md

```diff
@@ -0,0 +1,19 @@
+# Environment Variables
+
+Environment variables that can be used to configure `azd` behavior. For environment variables that accept a boolean, the values `1, t, T, TRUE, true, True` are accepted as "true"; the values: `0, f, F, FALSE, false, False` are all accepted as "false".
+
+- `AZD_ALPHA_ENABLE_<name>`: Enables or disables an alpha feature. `<name>` is the name of the feature, with dot `.` characters replaced by underscore `_` characters.
+- `AZD_AUTH_ENDPOINT`: The [External Authentication](./external-authentication.md) endpoint.
+- `AZD_AUTH_KEY`: The [External Authentication](./external-authentication.md) shared key.
+- `AZD_BUILDER_IMAGE`: The builder docker image used to perform docker-less builds.
+- `AZD_CONFIG_DIR`: The file path of the user-level configuration directory.
+- `AZD_DEMO_MODE`: If true, enables demo mode. Currently, this hides subscription IDs from being displayed in output.
+- `AZD_FORCE_TTY`: If true, forces `azd` write terminal-style output.
+- `AZD_IN_CLOUDSHELL`: If true, `azd` runs with cloud-shell specific behavior.
+- `AZD_SKIP_UPDATE_CHECK`: If true, skips the out-of-date update check output that is typically printed at the end of the command.
+
+For tools that are auto-acquired by `azd`, you are able to configure the following environment variables to use a different version of the tool installed on the machine:
+
+- `AZD_BICEP_TOOL_PATH`: The Bicep tool override path. The direct path to `bicep` or `bicep.exe`.
+- `AZD_GH_CLI_TOOL_PATH`: The `gh` tool override path. The direct path to `gh` or `gh.exe`.
```

Comment:
Should we standardize on `AZD_<tool invocation name>_TOOL_PATH`? `GH_CLI` is the outlier currently.

Reply:
I think that makes sense - will make this easier for us to programmatically generalize in the future.

Reply:
Made the one-line code change associated with this doc change.